# Content-related messages

content.content_missing=The node''s content is missing: \n   node: {0} \n   reader: {1}  \n   Please contact your system administrator.
content.runtime_exec.property_moved=The property ''errorCodes'' has moved down onto the RuntimeExec class

index.recovery.out_of_date=Os índices não estão sincronizados com a base de dados.
index.tracking.starting=Recuperação dos índices iniciada.
index.tracking.complete=Recuperação dos índices completa.
index.tracking.progress=\tProcessando transacções a {0}.
index.recovery.starting=Recuperação dos índices iniciada: {0} transacções.
index.recovery.complete=Recuperação dos índices completa.
index.recovery.progress=\t{0} % completado.
index.recovery.terminated=Recuperação dos índices terminada.

node.archive.msg.busy=Uma operação de recuperação ou limpeza em massa encontra-se em progresso

content.http_reader.err.no_connection=Não é possível ligar ao servidor remoto de Alfresco via HTTP: {0}
content.http_reader.err.no_authentication=O leitor de HTTP não foi capaz de autenticar no servidor remoto: {0}  \n
content.http_reader.err.check_cluster=Please ensure that 'replicateUpdates' and 'replicateUpdatesViaCopy' is enabled for the cache 'org.alfresco.cache.ticketsCache'.  Check that the general cluster configuration is correct and working.
content.http_reader.err.unrecognized=An unrecognized error occurred when attempting to download content from remote server:\n   Server: {0} \n   Content: {1} \n   HTTP Response: {2}

metadata.extraction.err.type_conversion=Metadata extraction failed because an extracted value failed to convert to the required type: \n   Extractor: {0} \n   Target Property QName: {1} \n   Required Type: {2} \n   Extracted Value: {3}

transform.err.format_or_password=Failed to convert content, possibly due to an incorrectly formatted or password protected file.

content.routing.err.invalid_default_store=The ''defaultStoreName'', ''{0}'' does not refer to a store in ''storesByName'' ({1}).